DigitalLearningSolutions.Web/Views/NotificationPreferences/Index.cshtml

```diff
@@ -21,6 +21,24 @@
           </span>
         </summary>
         <div class="nhsuk-details__text">
+          <dl class="nhsuk-summary-list preference-list">
+            @foreach (var notification in Model.AdminNotifications) {
+              <div class="nhsuk-grid-row preference-row">
+                <div class="nhsuk-grid-column-three-quarters">
+                  <div class="nhsuk-heading-xs">@notification.NotificationName</div>
+                  <p>@Html.Raw(notification.Description)</p>

```

Comment:
I'm always suspicious when I find a call to raw HTML, but this comes one-way out of the database so as long as what goes into that is properly controlled, we should be fine.

Reply:
The users cannot change the notification data so I think that's ok. On another note - could we specify the font size on the paragraph too? (I think NHS has body font classes in addition to the heading ones)